src/Microsoft.DotNet.Wpf/src/Shared/MS/Internal/Xaml/Parser/SpecialBracketCharacters.cs

```diff
@@ -20,7 +20,7 @@ internal class SpecialBracketCharacters : ISupportInitialize
     {
         private string _startChars;
         private string _endChars;
-        private readonly static ISet<char> _restrictedCharSet = new SortedSet<char>((new char[] { '=', ',', '\'', '"', '{', '}', '\\' }));
+        private readonly static ISet<char> _restrictedCharSet = new SortedSet<char>((new[] { '=', ',', '\'', '"', '{', '}', '\\' }));
```

Comment:
```suggestion
        private static readonly ISet<char> _restrictedCharSet = new SortedSet<char>(new[] { '=', ',', '\'', '"', '{', '}', '\\' });
```

1. You can remove redundant parentheses.
1. You can reorder the keywords.